### PR TITLE
refactor(test): Use helpers to interact with restmail.

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Use helpers to interact with restmail instead of accessing
restmail directly.

@mozilla/fxa-devs - r?

### CODE REVIEW

**Comment by shane-tomlinson** [comment]
Ref #4926

File: tests/functional/lib/helpers.js [hunk 1]
```diff
@@ -448,6 +448,29 @@ define([
...
+   *   @param {number} [options.maxAttempts] - number of email fetch attempts
+   *   to make. Defaults to 10.
+   * @returns {promise} resolves with the email headers if email is found.
+   */
+  const getEmailHeaders = thenify(function(user, index, options) {
+    if (/@/.test(user)) {
+      user = TestHelpers.emailToUser(user);
+    }
+
+    // restmail takes an index that is 1 based instead of 0 based.
```
Comment (philbooth) [inline tests/functional/lib/helpers.js]: I was a bit confused by this comment so went to look at the code. Would it be more accurate to say that restmail takes a length, hence adding 1 to the index?

Comment (philbooth) [inline tests/functional/lib/helpers.js]: (sorry, just seen that the comment is left over from old code below, still might be worth clarifying though)

**Comment by philbooth** [comment]
LGTM, r+

**Pull request merged**